.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thank you for your contribution! Please review https://microsoft.github.io/FLAML/docs/Contribute before opening a pull request. -->
+
+<!-- Please add a reviewer to the assignee section when you create a PR. If you don't have the access to it, we will shortly find a reviewer and assign them to your PR. -->
+
+## Why are these changes needed?
+
+<!-- Please give a short summary of the change and the problem this solves. -->
+
+## Related issue number
+
+<!-- For example: "Closes #1234" -->
+
+## Checks
+
+- [ ] I've used [pre-commit](https://microsoft.github.io/FLAML/docs/Contribute#pre-commit) to lint the changes in this PR, or I've checked [lint with flake8](https://github.com/microsoft/FLAML/blob/816a82a1155b4de4705b21a615ccdff67c6da379/.github/workflows/python-package.yml#L54-L59) output is two 0s.
+- [ ] I've included any doc changes needed for https://microsoft.github.io/FLAML/.
+- [ ] I've made sure the tests are passing.
+- Testing Strategy
+   - [ ] Unit tests
+   - [ ] This PR is not tested :(
```

Comment:
What do you think about replacing this with:
- [ ] If relevant, added tests corresponding to the changes introduced in this PR.
- [ ] Tests run successfully.

Reply:
Updated.